analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -1501,11 +1479,11 @@ void WhileFalse()
                 {
                     Console.WriteLine();
                 }
-                while (ConstantExpressionsAreExcluded.F);           // Noncompliant
+                while (ConstantExpressionsAreExcluded.F);           // Compliant it's a constant
             }
             void Condition()
             {
-                var x = ConstantExpressionsAreExcluded.T ? 1 : 2;   // Noncompliant
+                var x = ConstantExpressionsAreExcluded.T ? 1 : 2;   // FP maybe?
```

Comment:
I don't know if this test has no meaning anymore or if this should be considered an FP.
re-seeing this today I'd say it's a compliant case.

Reply:
It's a constant => compliant.